ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -15,14 +15,17 @@ import { Typography, Stack, Button, Box, useTheme, useMediaQuery, Alert } from '
 import PencilIcon from 'mdi-material-ui/PencilOutline';
 import AddPanelGroupIcon from 'mdi-material-ui/PlusBoxOutline';
 import AddPanelIcon from 'mdi-material-ui/ChartBoxPlusOutline';
-import { ErrorBoundary, ErrorAlert } from '@perses-dev/components';
+import { ErrorBoundary, ErrorAlert, InfoTooltip, TooltipPlacement } from '@perses-dev/components';
 import { DashboardResource } from '@perses-dev/core';
 import { useState } from 'react';
 import { useDashboard, useDashboardActions, useEditMode } from '../../context';
-import { TemplateVariableList } from '../Variables';
+import { TemplateVariableList, EditVariablesButton } from '../Variables';
 import { TimeRangeControls } from '../TimeRangeControls';
 import { DownloadButton } from '../DownloadButton';
 
+const ADD_PANEL_BUTTON_DESCRIPTION = 'Add panel';
```

Comment:
Curious, what's the rational of making these constants instead of defining the values inline where it's used?

Reply:
it's cuz I was using the same copy for the tooltip AND the aria-label, but open to changing it back 

I have another PR coming related to all the copy for all the tooltips on all the buttons, I wonder what you'll think (I wonder if my upcoming PR seems like over-engineering or actually helpful)

Reply:
Ah my bad, missed that!